.readthedocs.yaml

```
@@ -0,0 +1,24 @@
+# Required
+version: 2
+
+# Set the OS, Python version, and other tools you might need
+build:
+  os: ubuntu-24.04
+  tools:
+    python: "3.13"
+
+# Build documentation with Mkdocs
+mkdocs:
+   configuration: mkdocs.yml
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 3

(indentation)